resources/kubernetes/portal-api/portal-api.ts

```diff
@@ -15,8 +14,8 @@ const config = new pulumi.Config('portal-api');
 const authSignSecret = config.requireSecret('auth-sign-secret');
 const cookieSecret = config.requireSecret('cookie-secret');
 
-export const portalApiDevDomain = config.require('dev-domain');
-const cleanPortalApiDomain = portalApiDevDomain.slice(0, -1);
+export const portalApiDomain = config.require('domain');
```

Comment:
If we have a root domain (`fpx.no`), do we need to also get this value? 👀 that would be `api.fpx.no` right? We could just apply api as `api.{rootDomain}.no`?